### DISCUSSION TITLE
Migrating repositories for developers

### DISCUSSION THREAD

**Discussion by pandatix** [opentofu/opentofu#854] [General]
Hi everyone, as a provider dev I am looking since the very beginning to get the fork of [the terraform-plugin-framework](https://github.com/hashicorp/terraform-plugin-framework) in the OpenTofu ecosystem, but has not been yet.
Is it planned to fork it ? Or is the scaffolding framework the suggested way to do it in OpenTofu ? Or am I missing anything ?

Thanks in advance :)

**Comment by kislerdm** [comment]
@pandatix Hey Lucas! Thanks for your question! 

We have no plans of forking the providers framework at this point because its license remains unchanged (MPL 2.0). The scaffolding framework is still recommended way of developing providers. Please let us know in case of further questions.

**Reply by pandatix** [reply]
Hi, thank for this quick answer :)

**Reply by kislerdm** [reply]
@pandatix glad to help! 🙂
